packages/cli/api-importers/openapi-to-ir/src/3.1/OpenAPIConverter.ts

```diff
@@ -127,11 +128,12 @@ export class OpenAPIConverter extends AbstractConverter<OpenAPIConverterContext3
         });
 
         for (const [id, schema] of Object.entries(this.context.spec.components?.schemas ?? {})) {
+            const partiallyResolvedSchema = await this.resolveExternalRefs({ spec: schema, context: this.context });
```

Comment:
can we call `this.resolveExternalRefs` with the whole spec at the beginning of the convert function instead?